astroquery/hsa/tests/test_hsa_remote.py

```diff
@@ -0,0 +1,117 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
```

Comment:
These remote tests are very slow (I have let them run for 30mins, 2 failed one success, and cancelled the remaining), I'm not exactly sure about the reasons. If possible please use `obs_id`s that belongs to smaller files, etc.
While these tests are not running in CI for pull requests, we run them weekly from CI cron, and also run them manually to check for regression, it's really important that they would run in a reasonably quick timescale.

Reply:
I've changed the observations that were being downloaded to smaller ones and I've also changed the query so it only downloads one `product_level`. Now the tests take ~10 seconds.